requirements.txt

```diff
@@ -15,3 +15,4 @@ PyYAML
 typing; python_version < '3.5'
 enum34; python_version < '3.4'
 pathtools # supports vendored version of watchdog 0.9.0
+numpy
```

Comment:
Why are we making numpy a hard dependency?

Reply:
Used in redirect.py.

Reply:
Hard dependency removed. Falls back to naïve impl if numpy not available.